source/core/NumericString.cpp

```diff
@@ -559,33 +582,49 @@ void Format(SubString *format, Int32 count, StaticTypeAndData arguments[], Strin
                             if (precision < 0) {
                                 precision = 0;
                                 truncateSignificant = exponent + 1 - fOptions.Significant;
-                                // need truncate the integer part of the float because the sprintf doesnt do this for us.
+                                // need truncate the inxteger part of the float because the sprintf doesnt do this for us.
```

Comment:
Oops.  Fixed.  I noticed this when I previewed by diff but glossed over it because I thought I had fixed the typo rather than introducing it.